src/http/override.ts

```diff
@@ -98,7 +97,7 @@ export const overrideHttpModule: ModuleOverride = (middleware) => {
     }
 
     patchedModules[protocol] = {
-      module,
+      module: module,
```

Comment:
Why do you unfold this property assignment?

```js
{ key }
{ key: key }
```

Are identical. I think we should prefer the shorthand method.

Reply:
Agreed, I ran into some weird issues while debugging at first, figured `module` might be a reserved keyword. It turns out it isn't but I forgot to change this one back.